### DISCUSSION TITLE
Thank you VERY much!

### DISCUSSION THREAD

**Discussion by dakur** [tinymce/tinymce#8207] [General]
Just wanted to say thank you million times for such docs improvement!! Now (even though there are still some uncovered things) it's really pleasure to search for something there – mainly the algolia thing, but also improved design – font, offsets etc. The experience is completely different than in earlier versions! And together with the move to typescript, it's now *much* easier to upgrade or implement things.

Thank you very much once again to all of the team! ☺

**Comment by lostkeys** [comment]
Thank you for your kind words!
